great-idea/css/index.css

```diff
@@ -63,4 +63,25 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body{
+	max-width: 880px;
+	margin:0px auto;
+}
+.container { display: flex; 
+	justify-content: space-evenly;
+}
+.para1 { display: flex; flex-direction: row;
+	 justify-content: flex-end; width: 800px;
+	  border: 1px ; padding: 4px;}
```

Comment:
This still works but is pretty messy to read for other devs, and yourself in the future.